packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/GlobalSendReceive/GlobalSendReceive.tsx

```diff
@@ -10,11 +14,24 @@ import { GlobalSendReceiveButtons } from './GlobalSendReceiveButtons';
 
 export const GlobalSendReceive = () => {
     const { device } = useDevice();
+    const dispatch = useDispatch();
     const goToWithAnalytics = useGoToWithAnalytics();
     const buttonVariant = device?.connected && device?.available ? 'primary' : 'tertiary';
     const [isSendModalOpen, setIsSendModalOpen] = useState(false);
     const [isReceiveModalOpen, setIsReceiveModalOpen] = useState(false);
 
+    const prepareTokenSend = (account: Account, tokenAddress: TokenAddress) => {
```

Comment:
this is pretty weird, I guess ti would be better to introduce finally query parameters to the urls, so I could provice some default value to the send form